tests/unit/test_mms_messages.py

```diff
@@ -30,4 +30,4 @@ def test_send(self):
     def checkMessage(self, mmsMessage):
         assert mmsMessage.body == 'This is MMS sent from Zang'
         assert mmsMessage.status == MmsMessageStatus.QUEUED
-        assert mmsMessage.direction == MmsDirection.OUTBOUND_API
```

Comment:
For these enums - mms/sms/calls direction/status, we need to validate unify according to current values being used. please create a jira for this so that we'll keep track of this.